evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -146,9 +152,41 @@ pub fn apply<DB: Database>(
         HEVMCalls::Addr(inner) => addr(inner.0),
         HEVMCalls::Sign(inner) => sign(inner.0, inner.1.into(), data.env.cfg.chain_id),
         HEVMCalls::DeriveKey0(inner) => {
-            derive_key(&inner.0, DEFAULT_DERIVATION_PATH_PREFIX, inner.1)
+            derive_key::<English>(&inner.0, DEFAULT_DERIVATION_PATH_PREFIX, inner.1)
         }
-        HEVMCalls::DeriveKey1(inner) => derive_key(&inner.0, &inner.1, inner.2),
+        HEVMCalls::DeriveKey1(inner) => derive_key::<English>(&inner.0, &inner.1, inner.2),
+        HEVMCalls::DeriveKey2(inner) => match inner.2.as_str() {
```

Comment:
I'd like to have this as standalone function `derive_key_with_wordlist` that accepts the wordlist as str. perhaps even worth having an enum for this